Source/JavaScriptCore/assembler/X86Assembler.h

```diff
@@ -6391,9 +6391,15 @@ class X86Assembler {
         WTF::unalignedStore<int32_t>(ptr + 1, static_cast<int32_t>(distance));
     }
 
+    // FIXME: Indirection needed to work around GCC error in Debian 11.
+    static void* _memcpy(void* dest, const void* src, size_t n)
```

Comment:
We should name it like `memcpyWrapper`. `_` starting function is not allowed in C++ except for `std`.